### PR TITLE
Docs troubleshooting page

### PR DESCRIPTION
**Pull request by BOHEUS**
Related to #8296

### CODE REVIEW

**Comment by BOHEUS** [comment]
@Bonapara could you take a look? It's a WIP as I'm starting to check issues and gather all information

**Comment by Bonapara** [comment]
Looking good to me! I just can't remember if the docs supports H4 (####) did you check?

**Comment by BOHEUS** [comment]
From what I checked, H4 is supported like [here](https://twenty.com/developers/local-setup#cr-line-breaks-found-[windows])

File: packages/twenty-website/src/content/developers/self-hosting/qa.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+If file used for importing records has more than 20,000 records, then file won't be imported due to hardcoded limit.
+
+## Can I host Twenty in this provider?
+
+Some providers are verified by community and can be checked [here](https://twenty.com/developers/section/self-hosting/cloud-providers).
+If provider isn't listed in linked page but gives access to console, then it's likely possible to host Twenty.
+
+## I'd like to use Twenty at my company, but I'm missing this feature, when it'll be done?
+
+If feature you're looking for isn't on roadmap, you can create an issue or discussion with detailed explanation of requested feature and core team will review it and come back with answer.
```
[the remark this answers is not in the document]
Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/self-hosting/qa.mdx:31]: Once roadmap is created, link will be added

File: packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx [hunk 2]
```diff
@@ -45,7 +44,7 @@ Follow these steps for a manual setup.
 
     Copy the example environment file to a new .env file in your working directory:
     ```bash
-    curl -o .env https://raw.githubusercontent.com/twentyhq/twenty/main/packages/twenty-docker/.env.example
+    curl -o .env https://raw.githubusercontent.com/twentyhq/twenty/refs/tags/v0.32.0/packages/twenty-server/.env.example
```
[the remark this answers is not in the document]
Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx]: I don't like the way link in docker-compose page is presented with tag as it forces to update page every release but this secures from tickets about wrong configuration, how should it be solved?

**Comment by FelixMalfait** [comment]
Thanks! For the import limit error we should probably create an issue and improve the error message then? If the message was clear enough it the app we wouldn't need it here :)

**Comment by BOHEUS** [comment]
@FelixMalfait you're right, I'll take a closer look at it tomorrow and report issue after investigation

**Pull request merged**